packages/riipen-ui/src/components/MenuList.jsx

```diff
@@ -155,18 +155,11 @@ class MenuList extends React.Component {
   render() {
     const { children, classes, selectedIndex } = this.props;
 
-    const theme = this.context;
-
     return (
       <React.Fragment>
         <div className={clsx(classes)}>
           {this.getListItems(children, selectedIndex)}
         </div>
-        <style jsx>{`
-          div {
-            margin: ${theme.spacing(1)}px 0;
-          }
-        `}</style>
```

Comment:
I'm guessing this fixes a spacing issue that didn't exist before?

Reply:
![Screen Shot 2020-01-27 at 10 47 03 AM](https://user-images.githubusercontent.com/4382804/73203682-604ace80-40f2-11ea-8972-3fe8067ee563.png)
It fixes this tiny annoyance :)